### PR TITLE
Add ancestor images info in view

### PR DESCRIPTION
**Pull request by demoulinv**
## Description
This PR adds information in a view about ancestor images of this view. This is useful for example when an HDR image is created from a set of LDR images. In a view associated to the new HDR image, we store all information about all LDR images.
With those changes, sfmData version is upgraded to 1.2.6.

Corresponding Meshroom PR: https://github.com/alicevision/Meshroom/pull/2242

## Features list

- [X] View update (including deep copy)
- [X] Json importer / exporter update
- [X] Alembic importer / exporter update
- [X] Unitary test update

### CODE REVIEW

File: src/aliceVision/sfmData/SfMData.cpp [hunk 1]
```diff
@@ -42,6 +42,19 @@ bool SfMData::operator==(const SfMData& other) const
             return false;
     }
 
+    // Ancestors
+    if (_ancestors.size() != other._ancestors.size())
+        return false;
+
+    for (ImageInfos::const_iterator it = _ancestors.begin(); it != _ancestors.end(); ++it)
+    {
+        const ImageInfo& ancestor1 = *(it->second.get());
```
Comment (servantftransperfect) [inline src/aliceVision/sfmData/SfMData.cpp]: .get() are useless as shared_ptr are overloading the * operator

File: src/aliceVision/hdr/brackets.cpp [hunk 2]
```diff
@@ -273,10 +273,11 @@ int selectTargetViews(std::vector<std::shared_ptr<sfmData::View>>& out_targetVie
 
     for (auto& group : groups)
     {
-        // Set the ldr ancestors id
+        // Set the ldr ancestors
         for (auto v : group)
         {
             group[targetIndex]->addAncestor(v->getViewId());
+            group[targetIndex]->addAncestorImage(v->getViewId());
```
Comment (servantftransperfect) [inline src/aliceVision/hdr/brackets.cpp]: There is no need to keep both. They are the same.

Comment (fabiencastan) [inline src/aliceVision/hdr/brackets.cpp]: I think, we need only one. addAncestor should be enough.

Comment (demoulinv) [reply src/aliceVision/hdr/brackets.cpp]: Correction done.

**Review by servantftransperfect — DISMISSED** [review]

File: src/aliceVision/sfmData/SfMData.cpp (hunk 1 above)
Comment (cbentejac) [reply src/aliceVision/sfmData/SfMData.cpp]: Done.

**Review by cbentejac — APPROVED** [review]

**Pull request merged**